react-native-libraries.json

```diff
@@ -13083,5 +13083,15 @@
     "android": true,
     "web": true,
     "expoGo": true
+  },
+  {
+    "githubUrl": "https://github.com/cmpayc/react-native-fs-turbo",
+    "npmPkg": "react-native-fs-turbo",
+    "examples": ["https://github.com/cmpayc/react-native-fs-turbo?tab=readme-ov-file#examples"],
```

Comment:
```suggestion
```

Clicking the library name will lead user to it's GitHub repository, so there is no need to link to readme. Usually in this array we expect full example apps/projects or Snacks:
* https://github.com/react-native-community/directory?tab=readme-ov-file#examples